build.sbt

```diff
@@ -25,9 +24,15 @@ lazy val root = project
   .settings(noPublishSettings)
 
 lazy val core = myCrossProject("core")
+  .settings(
+    crossScalaVersions += "2.13.0"
+  )
   .settings(
     libraryDependencies ++= Seq(
-      Dependencies.cron4s,
+      scalaVersion.value match {
+        case "2.13.0" => Dependencies.cron4s13
+        case _ => Dependencies.cron4s12
+      },
```

Comment:
this ugly code is to allow 2.11 to use an older version than the one that 2.13 uses, since they dropped 2.11 support when they added 2.13.

Reply:
I'm okay with dropping support for 2.11 here too. So I'm fine with depending only on one version of cron4s.

Reply:
Dropped 2.11